snippets/Responsive-Nuemorphism-Card-Design.md

```diff
@@ -0,0 +1,176 @@
+
+---
+title: Responsive Nuemorphism Card Design
+tags: interactivity,layout
+---
+
+Creates an awesome nuemorphism effect with cool interactivity to the users. 
+
+- Uses `display: flex` property along with `justify-content: center; align-items: center;` to make the snippet responsive.
+- Creates Nuemorphism effect by using nuemorphism colors like `rgba(255, 255, 255, 0.1)` or `rgba(0, 0, 0, 0.6)` and using nuemorphism shadows. 
+- Uses `:hover` pseudo-selector to make the transitions and interactivity.
+
+```html
+<div class="container">
+  <div class="card">
+    <div class="box">
+      <div class="content">
+        <h2>01</h2>
+        <h3>Card One</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, totam velit? Iure nemo labore inventore?</p>
+        <a href="#">Read More</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="card">
```

Comment:
I think we could only have one card, otherwise the snippet is extremely long.

Reply:
> I think we could only have one card, otherwise the snippet is extremely long.

> Ok